cmd/frontend/internal/httpapi/webhookhandlers/handle_repo_authz_event.go

```diff
@@ -0,0 +1,55 @@
+package webhookhandlers
+
+import (
+	"context"
+	"fmt"
+	gh "github.com/google/go-github/v28/github"
+	"github.com/inconshreveable/log15"
+	"github.com/sourcegraph/sourcegraph/cmd/frontend/backend"
+	"github.com/sourcegraph/sourcegraph/cmd/repo-updater/repos"
+	"github.com/sourcegraph/sourcegraph/internal/actor"
+	"github.com/sourcegraph/sourcegraph/internal/api"
+	"github.com/sourcegraph/sourcegraph/internal/conf"
+	"github.com/sourcegraph/sourcegraph/internal/repoupdater"
+	"github.com/sourcegraph/sourcegraph/internal/repoupdater/protocol"
+)
+
+// handleGithubRepoAuthzEvent handles any github event containing a repository field, and enqueues the contained
+// repo for permissions synchronisation
```

Comment:
nit:
```suggestion
// repo for permissions synchronisation.
```

Docstrings for signatures should be sentences (with periods).